### DISCUSSION TITLE
Make the BossBar stay after player relogs

### DISCUSSION THREAD

**Discussion by RegencySolid** [Zrips/CMI#2236] [Feature Requests]
**Description of feature/improvement request:** When using timed boss bars, they disappear after relog even if they may have time left.  Make the boss bar reappear after login.

---
**Detailed explanation of what it should do, when and in what situation:** with the correct information shown, for example; you use a x2 exp voucher for mcmmo for 30 minutes, the boss bar has a count down from 30 minutes, you log out when its at 20 minutes for 10 minutes and when you log back in it's at 10 minutes and still displayed on your screen instead of disappearing when you logout.

**Basic explanation as to why this feature is needed and/or useful to have:** Having this feature would make timing a lot easier to display on the boss bar since it would constantly be displayed instead of disappearing on relog.

**Comment by mercurialmusic** [comment]
Yeah. I use it for restart warnings, and it would be nice to let people know if they're just logging in after the bossbar command was already run. I imagine this would be a fair bit more resource-intensive, though, and if that's the case I'm perfectly fine without it.

**Comment by Zrips** [comment]
Well we would need to store extra data about who should see particular boss bar messages and until when. it is possible and wont take a lot of server resources, if any at all. But some extra coding is needed for it to work.
